### ISSUE TITLE
How to support spring?

### ISSUE THREAD

**Issue by loveslikey** [wisdom-framework/wisdom#556]
Dose wisdom support spring framework?  How can I do?

**Comment by Riduidel** [comment]
No, wisdom framework doesn't support Spring. however, most (if not all) common features of Spring framework for web usage are supported by Wisdom Framework.
What are the specific elements of Spring Framework you would like to have in Wisdom Framework ?
